docs/sources/guides/whats-new-in-v6-6.md

```diff
@@ -158,6 +158,28 @@ Customize the login page, side menu bar, and footer links.
 
 {{< docs-imagebox img="/img/docs/v66/whitelabeling_1.png" max-width="700px" caption="White labeling example" >}}
 
+## Enterprise APT and YUM repositories
```

Comment:
It would make sense to add links to the install instructions for yum and debian repos next to the tables so that users that need a little bit more help can easily find it.